Sources/Tracking/CustomerIOImplementation.swift

```diff
@@ -177,7 +177,11 @@ public class CustomerIOImplementation: CustomerIOInstance {
         data: RequestBody,
         jsonEncoder: JSONEncoder? = nil
     ) {
-        trackEvent(type: .screen, name: name, data: data, jsonEncoder: jsonEncoder)
+        if sdkConfigStore.config.autoTrackScreenViews {
```

Comment:
this seems wrong - you should definitely be allowed to track screenviews regardless of whether automatic tracking is enabled

Reply:
Good catch. The automatic tracking is already taken care of when swizzling is enabled.